test_data_studies/studies/895/study_config.txt

```diff
@@ -0,0 +1,13 @@
+[required]
```

Comment:
You don't need to add this file or the other config file, these are created for each study.